src/components/Api.js

```diff
@@ -0,0 +1,117 @@
+// "https://around-api.en.tripleten-services.com/v1/"
+
+export default class API {
+  constructor(options) {
+    this._options = options;
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "eb144407-9d56-4c39-8eac-7fa32452a67f",
+      },
```

Comment:
According to the task you need to pass full `headers` and `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)